pom.xml

```diff
@@ -9,7 +9,7 @@
 
   <groupId>com.hotels</groupId>
   <artifactId>hotels-oss-plugin-config</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <version>1.1.2-SNAPSHOT</version>
```

Comment:
You can leave this as it was, after a release Maven automatically updates the version to the next candidate release version. The last release was 1.1.0 (you can refer to the CHANGELOG) so 1.1.1-SNAPSHOT is correct.